Jenkinsfile_CNP

```diff
@@ -43,28 +43,21 @@ def uploadCoreCaseDataDefinitions(env, version) {
 withPipeline("nodejs", product, component) {
   loadVaultSecrets(secrets)
 
-  onPR{
-    env.ENVIRONMENT="demo"
-    env.CCD_DEF_CLAIM_STORE_BASE_URL = "http://cmc-claim-store-demo.service.core-compute-demo.internal"
-  }
   onMaster{
     env.ENVIRONMENT="aat"
     env.CCD_DEF_CLAIM_STORE_BASE_URL = "http://cmc-claim-store-aat.service.core-compute-aat.internal"
   }
 
-  before('smoketest:preview') {
-    uploadCoreCaseDataDefinitions('demo', 'pr-615-3d45757c')
-  }
-
-  afterAlways('smoketest:preview') {
-    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'smoke-output/**'
-  }
-
   before('smoketest:aat') {
-    uploadCoreCaseDataDefinitions('aat', 'latest')
+    uploadCoreCaseDataDefinitions('aat', 'pr-615-3d45757c')
```

Comment:
is this still a thing?

Reply:
for some reason there isnt any latest. i check manually. ill look into it but it doesnt really matter as my other changes makes replaces this. 
https://github.com/hmcts/cmc-ccd-domain/pull/618/files#diff-a31d2a7e600e89b405cd675bd2f5bc6371e64b20cab9d32ab2a0279209a1882fR49

Reply:
just added it as im curious to see if this would work